### ISSUE TITLE
opencv with ffmpeg build fail

### ISSUE THREAD

**Issue by jasjuang** [Microsoft/vcpkg#2248]
`vcpkg install opencv[ffmpeg]  --featurepackages` yields

```
-- Warning: Could not find a matching pdb file for:
    C:/dev/vcpkg/packages/opencv_x64-windows/bin/opencv_ffmpeg331_64.dll
    C:/dev/vcpkg/packages/opencv_x64-windows/debug/bin/opencv_ffmpeg331_64.dll

-- Performing post-build validation
Detected outdated dynamic CRT in the following files:

    C:/dev/vcpkg/packages/opencv_x64-windows/debug/bin/opencv_ffmpeg331_64.dll: msvcrt.dll
    C:/dev/vcpkg/packages/opencv_x64-windows/bin/opencv_ffmpeg331_64.dll: msvcrt.dll

To inspect the dll files, use:
    dumpbin.exe /dependents mydllfile.dll
Found 1 error(s). Please correct the portfile:
    C:\dev\vcpkg\ports\opencv\portfile.cmake
-- Performing post-build validation done
Error: Building package opencv:x64-windows failed with: POST_BUILD_CHECKS_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencv:x64-windows
  Vcpkg version: 0.0.97-2017-11-22-22d015073f25ac10318cec47b58b6e405399aa7b

Additionally, attach any relevant sections from the log files above.
```

I remember this warning wasn't shown as error before, but some recent commit made it so.

**Comment by UnaNancyOwen** [comment]
I confirmed that this problem will be reproduced on my environment too.
I think this issue #942 is the same problem.
@ras0219-msft Could you fix this problem?

**Comment by UnaNancyOwen** [comment]
OpenCV with FFMPEG (<code>vcpkg install opencv[ffmpeg] --featurepackages</code>) will be able to install by applying the following changes to <code>vcpkg/toolsrc/src/vcpkg/postbuildlint.cpp</code>. 
(You need to rebuild vcpkg after changed using <code>vcpkg/bootstrap-vcpkg.bat</code>.)

[vcpkg/toolsrc/src/vcpkg/postbuildlint.cpp#L51](https://github.com/Microsoft/vcpkg/blob/master/toolsrc/src/vcpkg/postbuildlint.cpp#L51)
```patch
- {"msvcrt.dll", R"(msvcrt\.dll)"},
```

**Comment by jasjuang** [comment]
fixed by 4d274a7
